Dockerfile

```diff
@@ -1,6 +1,6 @@
 ##########################################################
 # Core Mava image
-FROM nvcr.io/nvidia/cuda:11.4.2-cudnn8-runtime-ubuntu20.04 as mava-core
+FROM registry.kao.instadeep.io/library/nvidia/cuda:11.4.2-cudnn8-runtime-ubuntu20.04 as mava-core
```

Comment:
Is this standard practice for us going forward? To use a InstaDeep-hosted image rather than an nividia one?